### DISCUSSION TITLE
呪いに多様性を与えたい

### DISCUSSION THREAD

**Discussion by dis-** [hengband/hengband#258] [Ideas]
現在、装備に付く永続呪いフラグは致命的なものと比較的無害なものの二極化が激しく中間層が不足している。
ギリギリ使用に堪える呪いを増やして表現の幅を増やしたい。

ざっと思いついた案
- 装備中、ターン経過でランダムにイベントが発生する
  - 騒音の発生(エキサイトモンスター)
  - 突然変異を得る (悪性変異を得やすい)
  - ときどき発火する/凍結する/電撃に撃たれる (物資破壊)
  - 恐怖する
  - 幻覚を見る
  - HPが減る
  - MPが減る
  - 記憶喪失
- 装備中、継続的に悪影響がある
  - じわじわステータスの現在値が減っていく
- 殴られたとき○○する
  - カオス的効果
  - バーサークする
  - テレポートする

他にも案を募集する。

**Comment by dis-** [comment]
- 永続
   - 少し深い階層のモンスターが出るようになる (日数経過+XX日相当)
   - 少しアイテム生成が悪くなる (黒いオーラ相当？)
   - 魔法成功率を下げる
   - 魔法道具成功率を下げる
   - 巻物/薬の使用エネルギーを少し増やす (+0.2ターン分くらいがやらしいのではなかろうか)

**Comment by Thujopsis** [comment]
こんなのどうでしょう。

- 特定の種族を呼び寄せる（特定シンボルの弱めのモンスター召喚）
- 逆祝福（祝福効果の逆転でマイナスを）
- 逆英雄化（ヒーロー化の逆転でマイナスを）
- アルコールの体内醸造（酔っ払いの薬の効果をランダムに）
- キノコの胞子を引き寄せる（ランダムにキノコの効果）

**Comment by deskull-m** [comment]
エゴの追加も捗りそうですな。MtG辺りからネーミング拝借してもいいかもしれない。

**Comment by ghost** [comment]
七つの大罪になぞらえていくつか考えてみましたが、嫉妬だけそれっぽいものが思いつきませんでした。
傲慢→隠密減少
強欲→ターン経過で所持金減
憤怒→ランダムに狂戦士化(魔法職で発動すると危険すぎる？)
色欲→常時エキサイトモンスター(女性@のみ？)
暴食→空腹度の進みが早くなる
怠惰→減速(-1～2程度？装備のメリットと天秤にかけられる程度のごく軽微な範囲)

**Comment by dis-** [comment]
装備している間。毎ターン特定の属性のダメージを受ける。オーラと同じく2d6くらいでいいか？
常用不可だが決戦時に頑張れば使える、元素ダメージなら免疫が映えるというデザインにしたい。

**Comment by daradarach** [comment]
重くなる。
武器だとクリティカル上がって強くなってしまうかも。解呪したくないジレンマとあわせてどうぞ的な。
防具はただただデメリット。

**Comment by aoi-Quylthulg** [comment]
徐々に切り傷や朦朧が蓄積していく
自然回復量より微妙に多い程度でも傷が治らなくなるから地味な厄介さを演出できるかも

**Comment by aoi-Quylthulg** [comment]
殺戮修正外での命中やダメージへのマイナス補正
低命中・高威力武器があっても命中を（+10）まで補強できてしまうので、それができないようにする

**Comment by daradarach** [comment]
浮遊をキャンセルする

**Comment by gold-onion** [comment]
PVALがプラスでも能力値修正は反転してマイナスになる
これで追加攻撃を持ちながら能力値修正がマイナスな武器を作れるようになる

**Comment by gold-onion** [comment]
攻撃が自分に当たる
死の大鎌の処理を流用
確率は1/3のままだと高すぎるかもしれない
